source/nodejs/adaptivecards-designer-app/src/app.ts

```diff
@@ -10,14 +10,10 @@ import "./app.css";
 // import "adaptivecards-designer/dist/adaptivecards-defaulthost.css";
 
 window.onload = function() {
-    // Uncomment to enabled preview features such as data binding
-    /*
```

Comment:
Sorry for jumping on this late @dclaux. But are we now enabling preview features by default on master with this uncommenting?

Reply:
Good catch - those are set to true so I can test. We need to decide what to do: the version picker should be enabled because this PR makes it a finished (as opposed to "preview") feature. Other flags should remain false I suppose.

But I'm not sure these changes should go to master just yet. Sent you a mail this morning about that.

Reply:
Agreed -- let's revert this to what it was until we can figure out an approach offline then? _The code itself landing in master is fine IMO; For publishing we could then discuss the environment - eg vnext perhaps before prod etc -- I am a bit hesitant exposing preview features as a default experience on our tooling lest we confuse our customers so thanks for starting that thread to kick start hte discussion! Thanks again!

Reply:
I am actually not sure the code landing in master is fine. Please refer to my email sent this morning, and let's take this offline.